posts/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from posts import views
+
+urlpatterns = [
+    url(r'^create/$', views.PostCreateView, name='create'),
+    url(r'^(?P<pk>\d+)/$', views.PostDetailView.as_view(), name='post'),
+
```

Comment:
Why? Isn't it necessary for [showing a single post](https://github.com/7Pros/circuit/pull/90/files#diff-866e65be3e11fd7705e62933a274314dR49)?